dockerfiles/init/modules/openshift/files/scripts/multi-user/deployPostgresAndKeycloak.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
could we avoid to use Uppercase in the name of the scripts ?
we have only `lower hyphen` or `lower_underscore`  sh files